hbase_storage/hbase1.2.0/src/main/java/com/splicemachine/access/hbase/HPartitionCreator.java

```diff
@@ -55,8 +55,9 @@ public PartitionCreator withName(String name){
 
     @Override
     public PartitionCreator withDisplayNames(String[] displayNames){
```

Comment:
Please update the corresponding HPartitionCreator for the other platforms (hbase1.0.0, hbase1.1.0...)

Reply:
Ok

Reply:
@dgomezferro Done, added to the other platforms.